### ISSUE TITLE
[stable/postgresql] Migrating existing postgresql chart to >=2.0.0 results (after manual process) in existing database unusable because of missing en_US locale

### ISSUE THREAD

**Issue by desaintmartin** [helm/charts#9232]
Migrating existing postgresql chart to >=2.0.0 results (after manual process) in existing database unusable because of missing en_US locale:
```
The database was initialized with LC_COLLATE "en_US.utf8",  which is not recognized by setlocale().
```

Fix: Running
```sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
    dpkg-reconfigure --frontend=noninteractive locales```

Works on existing pod. It would require a change in the Bitnami dockerfile. Would such change be acceptable?

Also, I've created a (complex) migration process, but I'm afraid it would be too complex for most people, do you want a WIP-PR for that?

cc @prydonius @tompizmor @sameersbn @carrodher @juan131

**Comment by desaintmartin** [comment]
Note: I realize I haven't followed the upgrade procedure (wrong version numbers in the readme), so I basically just have moved the postgres data directory and changed ownership and re-created a PVC. With this locale-gen change, it works well.

**Comment by desaintmartin** [comment]
Note: I I haven't followed the upgrade procedure because I use postgres as a subchart, so I basically just have moved the postgres data directory and changed ownership and re-created a PVC. With this locale-gen change, it works pretty well.

Uninstall and reinstalling the chart seems difficult in such cases (lot more downtime than just adapting postgres subchart).

**Comment by juan131** [comment]
We just released new images of PostgreSQL (**9.6.11-r3**, **10.6.0-r34**, **11.1.0-r29**) including the locales `en_GB.UTF-8` and `en_US.UTF-8` by default.

Please feel free to test these new images in the chart. The error should be fixed now.
